src/commands/force/user/create.ts

```diff
@@ -36,6 +37,31 @@ interface FailureMsg {
   message: string;
 }
 
+const permsetsStringToArray = (fieldsPermsets: string | string[]): string[] => {
+  if (!fieldsPermsets) return [];
+  return isArray<string>(fieldsPermsets)
+    ? fieldsPermsets
+    : fieldsPermsets.split(',').map((item) => item.replace("'", '').trim());
+};
+
+const standardizePasswordToBoolean = (input: unknown): boolean => {
+  if (typeof input === 'undefined' || input === null) {
```

Comment:
@mshanemc with default return now false, the undefined/null block can be removed, given remaining blocks are guarded with typeof checks.

Reply:
removed it, and also the check for the string `false` so that'll just hit default as well